services-directions/src/test/java/com/mapbox/api/directions/v5/MapboxDirectionsTest.java

```diff
@@ -179,6 +179,18 @@ public void build_originDestinationGetAddedToListCorrectly() throws Exception {
       .contains("1.234,2.345;13.493,9.958"));
   }
 
+  @Test
+  public void build_walkingOptions() {
+    MapboxDirections directions = MapboxDirections.builder()
+      .destination(Point.fromLngLat(13.4930, 9.958))
+      .origin(Point.fromLngLat(1.234, 2.345))
+      .accessToken(ACCESS_TOKEN)
+      .profile(DirectionsCriteria.PROFILE_WALKING)
+      .walkingOptions(WalkingOptions.builder().alleyBias(2d).build())
+      .build();
+    assertTrue(directions.cloneCall().request().url().toString().contains("alley_bias=2.0"));
```

Comment:
is it ok that this test is using a value that is out of bounds? valid values are from `-1` to `1`